stix2/observables.py

```diff
@@ -315,5 +347,5 @@ class X509Certificate(_Observable):
         'subject_public_key_algorithm': StringProperty(),
         'subject_public_key_modulus': StringProperty(),
         'subject_public_key_exponent': IntegerProperty(),
-        'x509_v3_extensions': Property(),
+        'x509_v3_extensions': EmbeddedObjectProperty(type=X509V3ExtenstionsType),
```

Comment:
Yes there are a few of those - mostly concerning extensions.  I will add them with the next changes.